lwt/server/Https.re

```diff
@@ -28,9 +29,15 @@ let make_request_handler: Server.t('s, 'r, 'a, 'b) => request_handler =
       let res = Httpaf.Response.create(status, ~headers);
       Httpaf.Reqd.respond_with_string(reqd, res, content);
     };
-    server
-    |> Server.middleware
-    |> Server.Middleware.run(closer, respond, req)
+    Request.read_body(reqd)
```

Comment:
Nitpick: I understand the module gets namespaced but to avoid confusion with the other `Request` modules, would it make sense to name it differently? Or is OCaml is it ok to have such cases?

Reply:
Is it okay in this case.